src/core/org/apache/jmeter/gui/MainFrame.java

```diff
@@ -680,7 +683,15 @@ private void addQuickComponentHotkeys(JTree treevar) {
 
             @Override
             public void actionPerformed(ActionEvent actionEvent) {
-                String propname = "gui.quick_" + actionEvent.getActionCommand();
+                //Bug 62336
+                AWTEvent current_event = EventQueue.getCurrentEvent();
```

Comment:
Have you checked what `actionEvent` is here? If I read the Swing-API correctly, it should be the same object as `current_event`. If `actionEvent` is really `null` here, is it only sometimes `null` and the NPE stops the AWT thread?
As a minor note: `current_event` should be written in camel case as `currentEvent` to match the other names.